client/post-editor/editor-status-label/index.jsx

```diff
@@ -65,6 +65,14 @@ export default React.createClass( {
 
 		statusClass = classNames( statusClass, 'is-' + this.props.post.status );
 
+		if ( ! this.props.onClick ) {
+			return (
+				<span className={ classNames( statusClass, 'is-plain' ) }>
```

Comment:
`is-plain` means that the status label is not clickable, and so it won't show the little chevron / cog icon at its left. In the editor tab, at the right of the site information, there is a status label with the class set to `is-plain`